apps/netsuite/connector.py

```
@@ -1162,7 +1162,42 @@ def sync_customers(self):
+            response = getattr(module, method)(internal_id)
+        else:
+            response = getattr(module, method)()
+
+        if method == 'get_all_generator':
+            response = [row for responses in response for row in responses]
+
+        return json.loads(json.dumps(response, default=str))
+
+    def get_exported_entry(self, resource_type: str, export_id: str):
+        module = getattr(self.connection, resource_type)
+        response = getattr(module, 'get')(export_id)
```

Ruff (0.8.0)

Do not call `getattr` with a constant attribute value. It is not any safer than normal property access.

Replace `getattr` with attribute access

(B009)